### PR TITLE
ENG-3150 postgres-image

### PR DESCRIPTION
**Pull request by smolino**

### CODE REVIEW

File: Dockerfile.centos8 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM centos:8
+
+RUN dnf install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+RUN dnf -qy module disable postgresql 
+RUN dnf install -y postgresql14-server
+RUN dnf install -y postgresql14-contrib
+USER postgres
+RUN /usr/pgsql-14/bin/initdb -D /var/lib/pgsql/14/data
+RUN echo "host all  all    0.0.0.0/0  md5" >> //var/lib/pgsql/14/data/pg_hba.conf
+RUN echo "listen_addresses='*'" >> /var/lib/pgsql/14/data/postgresql.conf
```
Comment (user-54dbb57a) [inline Dockerfile.centos8]: @smolino unify the"RUNS" in 2-3 commands, no need to create 7 different layers here

File: Dockerfile.ubi8 [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4-200
+
+RUN microdnf install -y dnf
+RUN dnf install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+RUN dnf install -y postgresql14-server
+RUN dnf install -y postgresql14-contrib
+USER postgres
+RUN /usr/pgsql-14/bin/initdb -D /var/lib/pgsql/14/data
+RUN echo "host all  all    0.0.0.0/0  md5" >> //var/lib/pgsql/14/data/pg_hba.conf
+RUN echo "listen_addresses='*'" >> /var/lib/pgsql/14/data/postgresql.conf
```
Comment (user-54dbb57a) [inline Dockerfile.ubi8]: @smolino unify the"RUNS" in 2-3 commands, no need to create 7 different layers here

**Review by user-54dbb57a — CHANGES_REQUESTED** [review]

File: Dockerfile.ubi8 [hunk 3]
```diff
@@ -0,0 +1,12 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4-200
+
+RUN microdnf install -y dnf && \
+         dnf install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm && \
+         dnf install -y postgresql14-server && \
+         dnf install -y postgresql14-contrib
+USER postgres
+RUN /usr/pgsql-14/bin/initdb -D /var/lib/pgsql/14/data && \
+         echo "host all  all    0.0.0.0/0  md5" >> //var/lib/pgsql/14/data/pg_hba.conf && \
```
Comment (user-54dbb57a) [inline Dockerfile.ubi8]: @smolino double slash at the start of the path

File: Dockerfile.centos8 [hunk 4]
```diff
@@ -0,0 +1,12 @@
+FROM centos:8
+
+RUN dnf install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm && \
+         dnf -qy module disable postgresql && \
+         dnf install -y postgresql14-server && \
+         dnf install -y postgresql14-contrib
+USER postgres
+RUN /usr/pgsql-14/bin/initdb -D /var/lib/pgsql/14/data && \
+         echo "host all  all    0.0.0.0/0  md5" >> //var/lib/pgsql/14/data/pg_hba.conf && \
```
Comment (user-54dbb57a) [inline Dockerfile.centos8]: @smolino double slash at the start of the path

File: entando-project [hunk 5]
```diff
@@ -0,0 +1,2 @@
+ENTANDO_PRJ_NAME=entando-postgres
+ENTANDO_PRJ_VERSION=14.0.0-SNAPSHOT
```
Comment (cecchisandrone) [inline entando-project]: this should reflect Postgres version, so I would say 14.1.0

File: Dockerfile.centos8 [hunk 6]
```diff
@@ -0,0 +1,17 @@
+FROM rockylinux:8.5
```
Comment (cecchisandrone) [inline Dockerfile.centos8]: Rename dockerfile as well

File: .env [hunk 7]
```diff
@@ -0,0 +1,3 @@
+POSTGRES_USER=
```
Comment (cecchisandrone) [inline .env]: these should be the supported variables

POSTGRESQL_USER
POSTGRESQL_PASSWORD
POSTGRESQL_DATABASE
POSTGRESQL_ADMIN_PASSWORD

**Review by cecchisandrone — APPROVED** [review]

**Review by user-54dbb57a — APPROVED** [review]

**Pull request merged**